packages/web/src/components/banner/DirectMessagesBanner.tsx

```diff
@@ -0,0 +1,51 @@
+import { useCallback, useState } from 'react'
+
+import { Client, accountSelectors } from '@audius/common'
+import { push as pushRoute } from 'connected-react-router'
+import { useDispatch } from 'react-redux'
+
+import { useSelector } from 'common/hooks/useSelector'
+import { getClient } from 'utils/clientUtil'
+import { CHATS_PAGE } from 'utils/route'
+
+import { CallToActionBanner } from './CallToActionBanner'
+
+const messages = {
+  text: 'Direct Messaging Now Available!',
+  pill: 'New'
+}
+
+const { getHasAccount } = accountSelectors
+
+const DIRECT_MESSAGES_BANNER_LOCAL_STORAGE_KEY = 'dismissDirectMessagesBanner'
+
+export const DirectMessagesBanner = () => {
```

Comment:
Let's add analytics for this banner; would like to see how many times it's seen as well as how many ppl click through/close it